LayoutTests/imported/w3c/web-platform-tests/webrtc/legacy/w3c-import.log

```diff
@@ -0,0 +1,22 @@
+The tests in this directory were imported from the W3C repository.
+Do NOT modify these tests directly in WebKit.
+Instead, create a pull request on the WPT github:
+	https://github.com/web-platform-tests/wpt
```

Comment:
Oh no, does this mean the `<!-- webkit-test-runner [ LegacyWebRTCOfferOptionsEnabled=true ] -->` preambles manually added will disappear at next import?

Reply:
The answer is no, thanks Ms2ger.